docs/.vitepress/config.js

```diff
@@ -56,7 +56,8 @@ function getMenuSidebar() {
         { text: 'Rock', link: '/playlist/rock' },
         { text: 'Soul', link: '/playlist/soul' },
         {text: 'Soundtrack', link: '/playlist/soundtrack'},
-        { text: 'Trap', link: '/playlist/trap' }
+        { text: 'Trap', link: '/playlist/trap' },
+        { text: 'Indie', link: '/playlist/indie' }
```

Comment:
Please sort alphabetically for add new genre inside that function, refer to this https://awesome-music-playlist.vercel.app/contribution.html#add-new-genre